permission_handler_windows/pubspec.yaml

```
@@ -0,0 +1,28 @@
+name: permission_handler_windows
+description: Permission plugin for Flutter. This plugin provides the Windows API to request and check permissions.
+version: 0.1.0
+homepage: https://github.com/baseflowit/flutter-permission-handler
```

nit: I know this is not your mistake but I realised this URL is wrong. We renamed our GitHub organisation from `baseflowit` to `baseflow` several years ago and only now noticed I have never updated the homepage here. Would you mind updating this to:
```suggestion
homepage: https://github.com/baseflow/flutter-permission-handler
```